wiki/Tournaments/Official_support/en.md

```diff
@@ -187,7 +187,7 @@ We have historically had issues where unscrupulous organisers have geared tourna
 
 ### Can a player who has been eliminated from play in my tournament join the staff after the fact and not cause problems? {#player-join-staff}
 
-No. Players who have been eliminated from the tournament/contest may not be enlisted as staff or assist in any organisational capacity with the sole exception of setting replays for mappool showcases. They can, however, be enlisted as a helper.
+Yes. Players who have been eliminated from a tournament may be enlisted as staff to assist as a referee, mappool selector, custom mapper, and/or a playtester.
```

Comment:
Our poll stated "every non-admin/non-managerial position". I think this covers different roles depending on the structure of the tournaments; perhaps it's better to phrase it the other way around?

```suggestion
Yes. Players who have been eliminated from a tournament may be enlisted as staff to assist to assist in any non-managerial position.
```

Reply:
Would work better indeed, wasn't too sure. Thanks!